frontend/src/styles/global.scss

```diff
@@ -558,13 +564,15 @@ body {
         --glass-bg-3000: var(--glass-bg-3000-light);
         --glass-border-3000: var(--glass-border-3000-light);
         --bg-light: #fff;
+        --bg-table: rgb(255 255 255 / 50%);
```

Comment:
This will cause problems for sticky columns. Don't know where I might reproduce it in the app but in Storybook:
<img width="678" alt="Screenshot 2023-11-17 at 15 12 05" src="https://github.com/PostHog/posthog/assets/6685876/dc24bb12-2805-43cd-a1fa-9681a8255aa0">

Reply:
Ah, then happy to set it to a hex color that's approximately representative. (Looks like `#F9FAF7` in light mode.)